ci/infra/testrunner/tests/test_skuba_upgrade.py

```diff
@@ -27,7 +27,7 @@ def setup_kubernetes_version(skuba, kubectl, kubernetes_version=None):
     kubectl.run_kubectl("wait --for=condition=ready nodes --all --timeout=5m")
 
 
-def test_upgrade_plan_all_fine(bootstrap, skuba, kubectl):
+def test_upgrade_plan_all_fine(skuba, kubectl):
```

Comment:
somehow Github still shows the old diff here :thinking: (without passing the `setup`)

the updated commit is https://github.com/MaximilianMeister/skuba/commit/d04f56f4d19609cca23c11d2140f14c17338451d